src/Themes/NexusMods.Themes.NexusFluentDark/Resources/Palette/CornerRadiuses/ElementCornerRadious.axaml

```diff
@@ -0,0 +1,29 @@
+﻿<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <!-- Values for CornerRadius property on Borders or other Components -->
+    
+    <!-- ReSharper disable InconsistentNaming -->
+    <CornerRadius x:Key="Rounded-none">0</CornerRadius>
+    <CornerRadius x:Key="Rounded-sm">2</CornerRadius>
+    <CornerRadius x:Key="Rounded">4</CornerRadius>
+    <CornerRadius x:Key="Rounded-md">6</CornerRadius>
+    <CornerRadius x:Key="Rounded-lg">8</CornerRadius>
+    <CornerRadius x:Key="Rounded-xl">12</CornerRadius>
+    <CornerRadius x:Key="Rounded-2xl">16</CornerRadius>
+    <CornerRadius x:Key="Rounded-3xl">24</CornerRadius>
+    <CornerRadius x:Key="Rounded-full">9999</CornerRadius>
```

Comment:
`9999`? What is `Rounded-full` supposed to represent?

Reply:
It means that the radius is the max that it can be, effectively circular. My understanding is that the actual radius will be:
`Min(Height/2, Width/2)` of the control